central/notifiers/jira/jira.go

```diff
@@ -376,12 +586,19 @@ func (j *jira) ProtoNotifier() *storage.Notifier {
 func (j *jira) createIssue(_ context.Context, severity storage.Severity, i *jiraLib.Issue) error {
 	i.Fields.Unknowns = j.unknownMap
 
-	if j.needsPriority {
+	if !j.conf.DisablePriority {
 		i.Fields.Priority = &jiraLib.Priority{
 			Name: j.severityToPriority[severity],
 		}
 	}
 
+	buf := new(bytes.Buffer)
+	err := json.NewEncoder(buf).Encode(i)
+	if err != nil {
+		return err
+	}
+	log.Debug(buf)
```

Comment:
Will this print a string?

Reply:
This is what I got in my log file from that statement:

```
notifiers/jira: 2024/01/08 22:30:49.063142 jira.go:600: Debug: {"fields":{"description":"StackRox Test Issue","issuetype":{"name":"Story"},"priority":{"name":"Low"},"project":{"key":"ROX"},"summary":"This is a test issue created to test integration with StackRox."}}
```